modules/odc-proc-binding-operator-backed-components-by-adding-a-servicebinding-resource.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
```

Comment:
@Dhruv: I see that both these files have same content:

- proc_binding-operator-backed-components-by-adding-a-servicebinding-resource.adoc
- odc-proc-binding-operator-backed-components-by-adding-a-servicebinding-resource.adoc

Suggest retaining any one and removing the other that is not relevant anymore.

Reply:
Fixed. Removing both as I have talked to Max and explained to him the reasons why the content should be kept in a single file. He has agreed to the changes and looks ahead to review it entirely.